src/po/Make_mvc.mak

```diff
@@ -123,22 +123,24 @@ ja.sjis.po: ja.po
 	@$(MAKE) -nologo -f Make_mvc.mak sjiscorr
 	-$(RM) $@
 !IF EXIST ("$(GETTEXT_PATH)\msgconv.exe")
-	"$(GETTEXT_PATH)\msgconv.exe" -t CP932 $? | .\sjiscorr.exe > $@
+	"$(GETTEXT_PATH)\msgconv.exe" -w 79 -t CP932 $? | .\sjiscorr.exe > $@
```

Comment:
here you are using `-w 79`, further down you are using `--width=`. Please use one consisten style. Also with regards to the quoation. you are now changing from `\"` . I'd prefer if you don't change unrelated things, it makes reviewing harder. If you want to change quotation style, add an additional commit just for those stylistic changes please.

Reply:
> here you are using `-w 79`, further down you are using `--width=`. Please use one consisten style.

For brevity, I wanted the line to fit into 80 columns. I'll fix it.

> Also with regards to the quoation. you are now changing from `\"` . I'd prefer if you don't change unrelated things, it makes reviewing harder. If you want to change quotation style, add an additional commit just for those stylistic changes please.

Oh, I'm sorry! You've already talked about it once. Let me not redo it this time. I'm sorry, please, once again.